s3/pkg/meta/db/drivers/tidb/bucket.go

```diff
@@ -309,38 +331,67 @@ func (t *TidbClient) ListObjects(ctx context.Context, bucketName, marker, verIdM
 					continue
 				}
 			}
+
 			var o *Object
-			Strver := strconv.FormatUint(version, 10)
-			o, err = t.GetObject(ctx, bucketname, name, Strver)
+			strVer := strconv.FormatUint(version, 10)
+			o, err = t.GetObject(ctx, bname, name, strVer)
 			if err != nil {
-				err = handleDBError(err)
+				log.Errorf("err:%v\n", err)
 				return
 			}
+
 			count += 1
 			if count == maxKeys {
 				nextMarker = name
```

Comment:
what is nextMarker used for ?

Reply:
It's the same as NextMarker used in aws s3,  comments is added now.